udata/core/dataset/models.py

```diff
@@ -824,8 +828,34 @@ def next_update(self):
         else:
             return self.last_update + delta
 
-    @cached_property
+    @property
     def quality(self):
```

Comment:
I wonder if keeping the existing `quality` cached_property but instead storing a `resource_quality` DictField on save wouldn't it make more readable. We wouldn't need a migration?

Reply:
Not sure how it solves our problem. In my new system, we don't need the migration (because we fallback to the computed version if there is no cache).

And not sure how `resource_quality` change something really major about the new system. Yes we could separate and have cache only for resources but it doesn't reduce the complexity? We still need a cache value, a computed value?